org.mixedrealitytoolkit.input/Controllers/MRTK3 Model XRI3.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// This component replaces the functionality of the deprecated XRController Model functionality to .
+    /// </summary>
+    /// <remarks>
+    /// TBD
+    /// </remarks>
+    public class MRTK3ModelXRI3 : MonoBehaviour
+    {
+        //TODO: Add Unity-test that check the MRTK3XRI3Model has the correct fields.
+
+        #region MRTK3 XRI 3 Model Properties
+
+        [SerializeField, Tooltip("The prefab of the MRTK Controller to show that will be automatically instantitated by this behaviour.")]
+        private Transform modelPrefab;
+
+        /// <summary>
+        /// The prefab of the MRTK Controller to show that will be automatically instantitated by this behaviour.
+        /// </summary>
+        /// <remarks>Expected to be XRNode.LeftHand or XRNode.RightHand.</remarks>
+        public Transform ModelPrefab => modelPrefab;
+
+        [SerializeField, Tooltip("The transform that is used as the parent for the model prefab when it is instantiated.  Will be set to a new child GameObject if None.")]
+        private Transform modelParent;
+
+        /// <summary>
+        /// The transform that is used as the parent for the model prefab when it is instantiated.  Will be set to a new child GameObject if None.
+        /// </summary>
+        /// <remarks>Expected to be XRNode.LeftHand or XRNode.RightHand.</remarks>
+        public Transform ModelParent => modelParent;
+
+        [SerializeField, Tooltip("The instance of the controller model in the scene.  This can be set to an existing object instead of using Model Prefab.")]
+        private Transform model;
+
+        /// <summary>
+        /// The instance of the controller model in the scene.  This can be set to an existing object instead of using Model Prefab.
+        /// </summary>
+        /// <remarks>Expected to be XRNode.LeftHand or XRNode.RightHand.</remarks>
+        public Transform Model => model;
+
+        #endregion MRTK3 XRI 3 Model Properties
+
+        #region Associated hand select values
+
+        [SerializeField, Tooltip("The XRNode associated with this Hand Controller. Expected to be XRNode.LeftHand or XRNode.RightHand.")]
+        private XRNode handNode;
+
+        /// <summary>
+        /// The XRNode associated with this Hand Controller.
+        /// </summary>
+        /// <remarks>Expected to be XRNode.LeftHand or XRNode.RightHand.</remarks>
+        public XRNode HandNode => handNode;
+
+        #endregion Associated hand select values
+
+        // Start is called before the first frame update
+        void Start()
+        {
+            if (!HandNode.Equals(XRNode.LeftHand) && !HandNode.Equals(XRNode.RightHand))
```

Comment:
I added it because of two reasons:
1 - It is part of the deprecated XRController Model functionality that needs to be replaced (Unity devs mentioned in [Slack](https://app.slack.com/client/E016WLPF0G6/C02GHQK9CP3?force_cold_boot=1) they don't plan to create a new home for it) as in:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/a47a01f7-7b2f-407e-a98c-4042cd024bed)
2 - Because we use it throughout the code via [ArticulatedHandController](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/main/org.mixedrealitytoolkit.input/Controllers/ArticulatedHandController.cs) as in:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/7d0231eb-51ca-4bad-ac41-87b76265ca22)

I thought on replicating it so that when obsolete classes/components are removed then editing its references would be easier.